__tests__/src/services/fetchErrorsHandler.spec.js

```diff
@@ -0,0 +1,26 @@
+import fetchErrorsHandler from '../../../src/services/fetchErrorsHandler';
+
+describe('HandleFetchErrors', () => {
+    test('should return a resolved promise containing the response if ok', async () => {
+        const response = {
+            ok: true,
+        };
+
+        expect.assertions(1);
```

Comment:
I find that line quite unnecessary. You test your test, it's 3 lines below. What's the point of this ?

```
> Please make sure I'm gonna ask you only 1 thing
> Asks one thing
> Was it OK ?
```

Reply:
I found this from the [doc](https://jest-bot.github.io/jest/docs/expect.html#expectassertionsnumber). It's a way to test asynchronous code.

> expect.assertions(number) verifies that a certain number of assertions are called during a test. This is often useful when testing asynchronous code, in order to make sure that assertions in a callback actually got called.

Reply:
Ok, you could have a global test saying `it should pass all assertions when called in [fetch|XHRHttpRequest|...] asynchronous context` instead of repeating it every time it's asynchronous.